templates/todo/projects/python-mongo/.repo/terraform/infra/loganalytics.tf

```diff
@@ -0,0 +1,18 @@
+# ------------------------------------------------------------------------------------------------------
+# Deploy log analytics workspace
+# ------------------------------------------------------------------------------------------------------
+resource "azurecaf_name" "workspace_name" {
+  name          = random_string.resource_token.result
+  resource_type = "azurerm_log_analytics_workspace"
+  random_length = 0
+  clean_input   = true
+}
+
+resource "azurerm_log_analytics_workspace" "workspace" {
+  name                = azurecaf_name.workspace_name.result
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  sku                 = "PerGB2018"
+  retention_in_days   = 30
+  tags                = local.tags
+}

```

Comment:
This may be a good candidate to add into the `appinsights` module? Maybe we rename the module to `logging` or `instrumentation` or similar?

Reply:
I would hold on combining for now, but something to consider for future.